### ISSUE TITLE
Feature: Integrate tuned/power-profile-deamon

### ISSUE THREAD

**Issue by baltevl** [Alexays/Waybar#3744]
Just spitballing an idea but tuned/ppd integration for the battery module would be cool. This could include showing a power profile in the tool tip and cycling through different profiles when clicked. If there is enough interest I'd be willing to create a pr :)
I'd prioritize tuned since it seems to be more feature rich but there are also arguments to be made for ppd as default. It seems like there is a compatibility layer for ppd from tuned 
 
resources:
https://wiki.archlinux.org/title/CPU_frequency_scaling#tuned
https://fedoraproject.org/wiki/Changes/TunedAsTheDefaultPowerProfileManagementDaemon
https://gitlab.freedesktop.org/upower/power-profiles-daemon#power-profiles-daemon
https://github.com/redhat-performance/tuned

**Comment by AlphaNecron** [comment]
`tuned-ppd` works fine, cycling through profiles does work as well.
![image](https://github.com/user-attachments/assets/2cc446a8-216c-42d4-937c-45445b4215e3)

**Comment by baltevl** [comment]
Thx for the response. Seams like i wasn't thorough enough when trying to figure this out. Going to close this now :)

**Comment by josecastillolema** [comment]
Hi!
Cycling though profile works well, however it does not get automatically updated.
I.e.: it shows power profile: performance when at the moment it has changed to:
```
> tuned-adm active
Current active profile: balanced
```

**Comment by silencespeakstruth** [comment]
I faced a similar issue too. The power-profiles-daemon module fails to appear in Waybar upon the initial launch of Hyprland. However, if Waybar is killed and restarted manually, the module appears and functions correctly. This suggests a race condition where Waybar attempts to connect to the D-Bus interface before the daemon has fully exposed its objects, despite the service being active.

I have already attempted to force the daemon to be ready before the graphical session starts by modifying the `systemd` unit (`power-profiles-daemon.service`) with the following overrides:

- Changed `After` targets and moved the service to basic.target to ensure it starts as early as possible.
- Added `ExecStartPost=/usr/bin/busctl wait-for-service net.hadess.PowerProfiles` to prevent the service from being marked as active until the D-Bus interface is actually responsive.

...but Waybar still fails to "see" the module during the initial cold start of the compositor. It seems the module lacks a retry mechanism or a reactive listener for the D-Bus interface appearing later in the session.
